src/Data/ProgrammesDb/Type/UtcDateTimeType.php

```diff
@@ -13,6 +13,8 @@
 
 class UtcDateTimeType extends DateTimeType
 {
+    private static $cache = [];
```

Comment:
A docblock about what this is... I dunno if `Chronos[][]` works or if it would be `array[]`